src/coreclr/src/vm/jithelpers.cpp

```diff
@@ -2569,7 +2211,7 @@ NOINLINE HCIMPL2(Object*, JITutil_ChkCastAny_NoCacheLookup, CORINFO_CLASS_HANDLE
 }
 HCIMPLEND
 
-NOINLINE HCIMPL2(Object*, JITutil_IsInstanceOfAny_NoCacheLookup, CORINFO_CLASS_HANDLE type, Object* obj)
+NOINLINE HCIMPL2(Object*, IsInstanceOfAny_NoCacheLookup, CORINFO_CLASS_HANDLE type, Object* obj)
 {
```

Comment:
It would be nice to add `HELPER_METHOD_POLL();` right before `HELPER_METHOD_FRAME_END` to these two NoCacheLookup helpers while you are on it. It will make sure that the casting helpers are 100% GC poll friendly (fixes https://github.com/dotnet/coreclr/issues/27931).